lib/widgets/HomeScreen/home_screen.dart

```diff
@@ -0,0 +1,87 @@
+import 'package:flutter/material.dart';
+import 'package:classroom_app/styles/color/colors.dart';
+import 'package:classroom_app/widgets/CustomButton/custom_button.dart';
+
+class HomeScreen extends StatelessWidget {
+  const HomeScreen({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    double screenWidth = MediaQuery.of(context).size.width;
+    double screenHeight = MediaQuery.of(context).size.height;
+    return Scaffold(
+      backgroundColor: AppColors.white,
+      body: Padding(
+        padding: EdgeInsets.symmetric(horizontal: screenWidth * 0.05),
+        child: Column(
+          mainAxisAlignment: MainAxisAlignment.center,
+          crossAxisAlignment: CrossAxisAlignment.center,
+          children: [
+            Center(
+              child: Image.asset(
+                'assets/images/overviewPage_image.png',
```

Comment:
If the asset is used frequently or on multiple screens, loading it repeatedly can be inefficient.
Use const for the Image.asset widget if the Image won't change or cache it using a const Widget.

```
const AssetImage('assets/images/overviewPage_image.png')
```

Reply:
The IDE does the feature for this, If there is a need to be a const keyword it suggests changing to const, if there is a const but not need to be const the IDE suggested to remove it.

Reply:
ditto

Reply:
This image is used only once. So, I want to keep it as it is if you agree.

Reply:
replace hardcoded image paths with assets_path.dart references